hummingbird/ml/convert.py

```diff
@@ -15,7 +15,7 @@
 from ._parse import parse_sklearn_api_model, parse_onnx_api_model, parse_sparkml_api_model
 from ._topology import convert as topology_converter
 from ._utils import (
-    torch_installed,
+    assert_torch_installed,
```

Comment:
Can you please also change the other `_installed` (Ex lightgbm/xgeboot/pandas) to be prefixed with `assert` for consistency?

Reply:
Done!

Reply:
I've also fixed some spelling errors along the way. In this case, the word `XGBoost` and `install` was incorrectly spelled and capitalized!
![image](https://user-images.githubusercontent.com/72343262/121113660-9ea2e280-c83c-11eb-85dc-c80a39dc594a.png)